.github/actions/derive-cilium-vsn/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Derive Cilium version
+description: Derive Cilium version
+inputs:
+  from_vsn:
+    description: Derive Cilium version for given major.minor version (e.g., v1.13) OR latest stable if not specified
+    required: true
+    default: ''
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      if: ${{ inputs.from_vsn != '' }}
+      run: |
+        VSN=$(curl -fsSL https://raw.githubusercontent.com/cilium/cilium/${{ inputs.from_vsn }}/VERSION)
+        echo "CILIUM_VSN=${VSN}" >> $GITHUB_ENV
```

Comment:
In K8sUpdates, we used to up/downgrade from/to the stable branch and not any specific release. That was convenient when we needed to patch the branch to make either the upgrade or the downgrade work; we didn't have to wait for the patch to be released before seeing the test passing.

Is the same not possible here?

Reply:
Ouch, this `from_vsn` naming might be a bit misleading. It should be read as _derive the latest stable version from a given vsn_. For example, `from_vsn=v1.13`, it will derive and set `env.CILIUM_VSN=v1.13.5`. Later on, the test does the following:

1. Install `env.CILIUM_VSN`, and test.
2. Upgrade to the PR built image, and test. 
3. Downgrade to `env.CILIUM_VSN`, and test.

Reply:
Ah, I see. Make sense. Then we can catch the regressions happening in backport branches too :+1:

Anyway, it seems that there is no easy way to instruct the CLI's install / upgrade to pick the latest unreleased v1.13. I will follow up on this. But first, let's merge this PR.

Reply:
Oh, found the way how to install. Please hold my bear, trying to do it in this PR :crossed_fingers: